specification/agrifood/data-plane/Microsoft.AgFoodPlatform/preview/2022-11-01-preview/agfood.json

```diff
@@ -12840,6 +12895,126 @@
         }
       }
     },
+    "/scenes/collections/{collectionId}/features/{featureId}": {
+      "get": {
+        "tags": [
+          "Scenes"
+        ],
+        "description": "Get a feature(STAC Item) for given collection and feature id.",
```

Comment:
Can we also include the name of STAC in the description? 

E.g. SpatioTemporal Asset Catalog (STAC) Item

Reply:
Added the whole name.